client/src/components/ElementActions/CustomActions.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+const CustomActions = (props) => {
+  const { children } = props;
+
+  // Don't render if there are no children
+  if (!Array.isArray(children) || !children.length > 0) {
+    return null;
+  }
+
+  return (
+    <div role="none" onClick={(event) => event.stopPropagation()}>
```

Comment:
This is annoyingly required as the div it sits within has an event listener which will try and navigate away from the page if we don't stop the event from propagating